src/main/scala/mesosphere/marathon/core/instance/Instance.scala

```diff
@@ -385,7 +385,29 @@ object Instance {
   }
 
   implicit val instanceConditionFormat: Format[Condition] = Condition.conditionFormat
-  implicit val instanceStateFormat: Format[InstanceState] = Json.format[InstanceState]
+
+  implicit val instanceStateWrites: Writes[InstanceState] = {
```

Comment:
Could you add a comment here that we explicitly drop the `goal` here until xyz is merged?

Reply:
The follow up with migrations is already up. I think that we'll merge both at the same time. So I don't think we need the comment.

Reply:
SGTM